advanced_usage/other_config_options.md

```diff
@@ -116,3 +116,27 @@ Go Server's config directory. The default value of this property is specified
 
 Changing this could have an impact on the ability to upgrade the Go Server, and
 so, it's not recommended to change this.
+
+<a name='go.spa.refresh.interval'></a>
+### go.spa.refresh.interval - GoCD SPA pages refresh interval time
+
+- Name: go.spa.refresh.interval
+- Type: [System property](#system-properties)
+- Restrictions: Should be the time duration specified in milliseconds
+- Default: 10000
+
+This property can be used to change the time interval at which GoCD pages should be refreshed.
```

Comment:
Wondering if this can be a little more elaborate, something like  


The SPA pages on GoCD like Agents are refreshed periodically to ensure they have upto date information. This property can be used to change the time interval at which these GoCD pages should be refreshed. Setting this to a lower value (<=5000) could have a performance impact on the GoCD server.